packages/gatsby/src/utils/webpack/utils/__tests__/create-normalized-module-key.ts

```diff
@@ -0,0 +1,40 @@
+import path from "path"
+import { createNormalizedModuleKey } from "../create-normalized-module-key"
+
+const projectRoot = path.join(`Users`, `username`, `project`)
+
+it(`should normalize bare module imports from node_modules`, () => {
+  const nodeModule = path.join(`node_modules`, `package-name`)
+
+  const normalizedModuleKey = createNormalizedModuleKey(
+    path.join(projectRoot, nodeModule),
+    projectRoot
+  )
+
+  expect(normalizedModuleKey).toMatchSnapshot()
```

Comment:
NIT: I'd suggest using `.toMatchInlineSnapshot()` here given that output is not large, and that will collocate "expected" value to be here, making it easier to see what assertion is actually being done here (regular snapshot requires looking stuff up in `.snap` files etc).

Not big deal, just my preference for this kind of things